webapp/app/js/controllers/cubeSchema.js

```diff
@@ -36,6 +36,8 @@ KylinApp.controller('CubeSchemaCtrl', function ($scope, QueryService, UserServic
     ];
 
     $scope.curStep = $scope.wizardSteps[0];
+    $scope.allCubeNames = [];
+    $scope.cubeService = CubeService;
```

Comment:
$scope.cubeService seems never used

Reply:
Hi， chunen：
Yes, "$scope.cubeService"  never be used. 
I will submit code to delete this line tonight.

Reply:
Hi,  chunen:
I have deleted this unused line: "$scope.cubeService = CubeService" and submit code.
Please verify this patch, Thank you.